Models/user.js

```diff
@@ -14,8 +14,9 @@ const model = mongoose.model("users", new Schema({
     },
     note: String,
   },
-  responders: [{_id:false, id: String}]
-}));
+  responders: [{_id:false, id: String}],
+  pushToken: Object
```

Comment:
Why is pushToken an Object is the request is a string?

Reply:
Good question. I initially thought the token would be an object but turns out to be something like ExponentPushToken[blahblahblah]. I will turn it back into a string type